tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	parser "github.com/kubevirt/monitoring/pkg/metrics/parser"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+var excludedMetrics = map[string]struct{}{
+	"kubevirt_ssp_operator_up_total":           {},
+	"kubevirt_ssp_template_validator_up_total": {},
+}
+
+func readMetrics() []*dto.MetricFamily {
+	var metricFamilies []*dto.MetricFamily
+	metrics := metrics.RecordRulesDescList
+
+	//metricsList := make([]parser.Metric, len(sspMetrics))
+	for _, metric := range metrics {
+		if _, isExcludedMetric := excludedMetrics[metric.Name]; !isExcludedMetric {
```

Comment:
Add comment // // Remove ignored metrics from all rules

Reply:
it doesn't remove the excluded metrics, it's just excludes them when we group to `metricFamilies`. I find this `if`  enough self-explained.

Reply:
I think that the fact that we excluded these metrics is self-explained, but from `all roles` is less self-explained.
But it's fine by me to leave as is.

Reply:
what do you mean by all rules?

Reply:
all rules means that it is excluded from all linter tests